setup.cfg

```diff
@@ -18,7 +18,7 @@ extend-ignore =
     F841  # Unused variables are checked by pylint
 
 [tool:pytest]
-addopts = -n auto -rfE -s --strict-markers
+addopts = -rfE -s --strict-markers
```

Comment:
please remember to remove this once you are satisfied -- if you don't use auto your tests won't run fast enough.